### ISSUE TITLE
ProtocolError (/var/task/src/handlers/pdf.js:3608:9)

### ISSUE THREAD

**Issue by Taylorsuk** [adieuadieu/serverless-chrome#125]
Awesome work with this project, everything deployed fine, however getting the following error in CloudWatch when running a PDF generation, this just gives a blank PDF in the browser  as a result. 

```
2018-03-19T18:35:12.122Z 416355ff-2ba4-11e8-b9d4-e55847aed77c { Error: Printing failed
at ProtocolError (/var/task/src/handlers/pdf.js:3608:9)
at /var/task/src/handlers/pdf.js:3680:30
at Chrome.handleMessage (/var/task/src/handlers/pdf.js:3881:13)
at WebSocket.<anonymous> (/var/task/src/handlers/pdf.js:3858:27)
at emitOne (events.js:96:13)
at WebSocket.emit (events.js:188:7)
at Receiver._receiver.onmessage (/var/task/src/handlers/pdf.js:1082:47)
at Receiver.dataMessage (/var/task/src/handlers/pdf.js:2407:14)
at perMessageDeflate.decompress (/var/task/src/handlers/pdf.js:2367:40)
at _decompress (/var/task/src/handlers/pdf.js:456:9) response: { code: -32000, message: 'Printing failed' } }
```

**Comment by jaypea** [comment]
Hi @Taylorsuk, did you resolve this issue? I'm seeing the same thing and could need a hint where to look. thanks.

**Comment by Taylorsuk** [comment]
@jaypea I recall getting it working but I cannot remember what I did TBH, it was just a world of pain. Then I found the function so unreliable that I looked around for a 3rd part service. I stumbled upon [ Prerender.Cloud](https://www.prerender.cloud/) which has a simple Node Library and offers good value for money. Run a good 1000+ PDFs through the service without a single error and good response times considering it's transiting the atlantic. I spoke with Jonathan the founder and there is an option for an EU server if it's finantially viable for him.
